packager/transformer.js

```diff
@@ -33,28 +42,22 @@ function transform(src, filename, options) {
 
   const config = Object.assign({}, babelRC, extraConfig);
 
+  // Add extra plugins
+  const extraPlugins = [require('babel-plugin-external-helpers')];
+
   if (options.inlineRequires) {
     extraPlugins.push(inlineRequires);
   }
   config.plugins = extraPlugins.concat(config.plugins);
 
-  // Manually resolve all default Babel plugins. babel.transform will attempt to resolve
-  // all base plugins relative to the file it's compiling. This makes sure that we're
-  // using the plugins installed in the react-native package.
-  config.plugins = config.plugins.map(function(plugin) {
-    // Normalise plugin to an array.
-    if (!Array.isArray(plugin)) {
-      plugin = [plugin];
-    }
-    // Only resolve the plugin if it's a string reference.
-    if (typeof plugin[0] === 'string') {
-      plugin[0] = require(`babel-plugin-${plugin[0]}`);
-      plugin[0] = plugin[0].__esModule ? plugin[0].default : plugin[0];
-    }
-    return plugin;
-  });
-
-  const result = babel.transform(src, Object.assign({}, babelRC, config));
+  return Object.assign({}, babelRC, extraConfig);
```

Comment:
Shouldn't `extraConfig` be `config` instead? `config` doesn't seems to be used anywhere.